src/NzbDrone.Core/Download/Clients/DownloadStation/Proxies/DiskStationProxyBase.cs

```diff
@@ -172,40 +112,62 @@ private string GenerateSessionCacheKey(DownloadStationSettings settings)
 
         private string AuthenticateClient(DownloadStationSettings settings)
         {
-            var arguments = new Dictionary<string, object>
-            {
-                 { "api", "SYNO.API.Auth" },
-                 { "version", "1" },
-                 { "method", "login" },
-                 { "account", settings.Username },
-                 { "passwd", settings.Password },
-                 { "format", "sid" },
-                 { "session", "DownloadStation" },
-             };
-
             var authInfo = GetApiInfo(DiskStationApi.Auth, settings);
 
-            var requestBuilder = BuildRequest(settings, authInfo, HttpMethod.GET);
+            var requestBuilder = BuildRequest(settings, authInfo, "login", 2);
+            requestBuilder.AddQueryParam("account", settings.Username);
+            requestBuilder.AddQueryParam("passwd", settings.Password);
+            requestBuilder.AddQueryParam("format", "cookie");
```

Comment:
cookie? wasn't this 'sid'?

PS: The use of AddQueryParam is exactly what I meant.